product-listing/products.css

```diff
@@ -0,0 +1,93 @@
+.middleSection{
+    margin-top: 80px;
+    overflow: hidden;
+    height: 89vh;
+    display: flex;
+}
+.sidebar{
+    overflow: auto;
+    padding: 0.5rem;
+    flex: 1 0;
+    flex-basis: 230px;
+    box-shadow: 1px 2px 2px 1px var(--secondary);
```

Comment:
you can create box-show variable, just suggestion.

Reply:
sure, thanks!